src/components/safe-apps/SafeAppIconCard/index.tsx

```diff
@@ -46,23 +16,7 @@ const SafeAppIconCard = ({
   height?: number
   fallback?: string
 }): ReactElement => {
-  if (_isSafeSrc(src)) {
-    return <ImageFallback src={src} alt={alt} width={width} height={height} fallbackSrc={fallback} />
-  }
-
-  return (
-    <iframe
-      title={alt}
-      srcDoc={getIframeContent(src, width, height, fallback)}
-      sandbox="allow-scripts"
-      referrerPolicy="strict-origin"
-      width={width}
-      height={height}
-      style={{ pointerEvents: 'none', border: 0 }}
-      tabIndex={-1}
-      loading="lazy"
-    />
-  )
+  return <ImageFallback src={src} alt={alt} width={width} height={height} fallbackSrc={fallback} />
```

Comment:
Wdyt about keeping the `_isSafeSrc` check? We wouldn't be able to detect if something changes and it could accidentally set cookies or other things for the user.

Reply:
Restored 👍